src/IO/HTTPCommon.cpp

```diff
@@ -99,11 +100,15 @@ Exception HTTPException::makeExceptionMessage(
     const std::string & reason,
     const std::string & body)
 {
+    std::string body_log_buffer;
+
     return Exception(code,
         "Received error from remote server {}. "
-        "HTTP status code: {} {}, "
-        "body: {}",
-        uri, static_cast<int>(http_status), reason, body);
+        "HTTP status code: {} '{}', "
+        "body: '{}'",
```

Comment:
Do we even need the body? It leads to dumping data into logs, which might be sensitive and I don't expect it's providing useful information anyway.

Reply:
Indeed. We can log body size, just to give a clue about the response